README.md

```diff
@@ -277,3 +277,31 @@ to your local platform and then build (Adopt) OpenJDK and place the result in
 
 These scripts default to using AdoptOpenJDK as the OpenJDK source repo to build 
 from, but you can override this with the `-r` flag.
+
+
+# Build farm
+
+The documentation for files that define the build farm builds can be found [here](/docs/build.md). 
```

Comment:
Usually better for SEO to have the link say `Build Farm Definition` as opposed to `here`